src/main.cpp

```diff
@@ -16,21 +16,27 @@ int main(int argc, char *argv[]) {
 		return 0;
 	}
 
+	if (argv[argc - 1] == std::string("-h")) {
+		Frontend::Helper(helpArgument);
+		return 0;
+	}
 	for (int i = 1; i < argc; ++i) {
 		if (file_included(argv[i])) {
 			filename = argv[i];
 		} else if (argv[i] == std::string("-o")) {
 			frontend.outputFileName = std::string(argv[++i]);
+		} else {
+			Frontend::Exception(argError, argv[i]);
+			return 0;
 		}
 	}
-
 	if (drv.parse_file(filename) == 0) {
 		frontend.drv = &drv;
 		frontend.WriteFile();
 	} else {
 		std::cout << "Compilation error" << std::endl;
 	}
-	Frontend::Helper();
-
+	
+        Frontend::Helper();
 	return 0;
```

Comment:
If it gets to the end, without returning 0 beforehand, it should probably return 1.

Reply:
I just fixed it.